aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -4267,39 +4267,22 @@ TORCH_IMPL_FUNC(linalg_ldl_factor_ex_out)
     return;
   }
 
-  auto pivots_ = pivots.expect_contiguous();
-  auto info_ = info.expect_contiguous();
-
-  auto LD_ = at::native::borrow_else_clone(
-      LD.mT().is_contiguous(), LD, self, /*row_major=*/false);
-  if (LD.mT().is_contiguous()) {
-    LD_->copy_(self);
-  }
+  LD.copy_(self);
```

Comment:
Now we can even use `triu_out`/`tril_out` below and put the correct thing into LD directly, which is niiice :D